source/css/sass/patterns/molecules/tag-list.scss

```diff
@@ -1,27 +1,34 @@
 @import "../../mixins/sizes";
 @import "../../mixins/typography";
 
-.tag-list {
+.tags {
   @include blg-spacing("block-end", "small", "margin");
 }
 
-.tag-list__title {
+.tags-list__title {
   @include label-content-typography();
   @include padding(0, "block");
 }
 
-.tag-list__list {
+.tags-list {
   list-style: none;
   @include blg-spacing("block-end", "small", "margin");
   @include label-tag-typography();
   @include padding(0, "inline-start");
 }
 
-.tag-list__list--single-line {
+.tags-list__single-line {
   @include truncate-with-ellipsis();
 }
 
-.tag-list__item {
+.tags-list__article-type {
+  span,
```

Comment:
We try to avoid using elements in selectors as it can make maintenance more difficult. Would a better selector here be `.tags-list__article-type .tags-list__link`?